### PR TITLE
docs: improve the chinese doc of install_deploy

### PR DESCRIPTION
**Pull request by lumianph**

### CODE REVIEW

File: docs/zh/deploy/install_deploy.md [hunk 1]
```diff
@@ -212,13 +160,11 @@ tar -zxvf openmldb-0.7.1-linux.tar.gz
...
 ```
 
-### 配置
-#### 环境配置
+### 环境配置
 环境变量定义在`conf/openmldb-env.sh`，如下表所示：
 
 | 环境变量                              | 默认值                                | 定义                                                                      |
 |-----------------------------------|------------------------------------|-------------------------------------------------------------------------|
-| OPENMLDB_VERSION                  | 0.7.1                              | OpenMLDB版本                                                              |
```
Comment (dl239) [inline docs/zh/deploy/install_deploy.md]: why delete the version

File: docs/zh/deploy/install_deploy.md [hunk 2]
```diff
@@ -247,9 +193,6 @@ node3:7527
 [apiserver]
 node3:9080
 
-[taskmanager]
-localhost:9902
```
Comment (dl239) [inline docs/zh/deploy/install_deploy.md:251]: dose not merge the latest upstream ?

Comment (zhanghaohit) [reply docs/zh/deploy/install_deploy.md:251]: this should change to `node3:9902`

`localhost` is not accessible outside its own node

File: docs/zh/deploy/install_deploy.md (hunk 1 above)
Comment (lumianph) [reply docs/zh/deploy/install_deploy.md]: this should be a typo... will recover this

File: docs/zh/deploy/install_deploy.md (hunk 2 above)
Comment (lumianph) [reply docs/zh/deploy/install_deploy.md:251]: @zhanghaohit pls double check

**Review by dl239 — APPROVED** [review]

**Review by zhanghaohit — APPROVED** [review]

**Pull request merged**